crates/sargon/src/wrapped_radix_engine_toolkit/low_level/manifest_summary/account_deposit/change_source.rs

```diff
@@ -0,0 +1,109 @@
+use crate::prelude::*;
+
+/// Created by the visitor, generally references a particular instruction,
+/// or maybe an initial YIELD_TO_PARENT.
+#[derive(Clone, Debug, PartialEq, Eq, uniffi::Enum)]
+pub enum ChangeSource {
+    InitialYieldFromParent,
```

Comment:
are there any documentation on each Variant (Case) inside of RET that you can carry over?

Reply:
No, there's only documentation for the enum